docker-compose-files/dspace-compose/sword.override.yml

```diff
@@ -4,12 +4,11 @@ version: "3.7"
 services:
   dspace:
     environment:
-      # Double underbars in env names will be replaced with periods for apache commons
       # __D__ -> -
       # __P__ -> .
       - sword__D__server__P__enabled=true
       - swordv2__D__server__P__enabled=true
       - swordv2__D__server__P__path=swordv2
-      - swordv2__D__server__P__url=http://localhost:8080/swordv2
-      - swordv2__D__server__P__collection__P__url=http://localhost:8080/swordv2/collection
-      - swordv2__D__server__P__servicedocument__P__url=http://localhost:8080/swordv2/servicedocument
+      - swordv2__D__server__P__url=http://localhost:8080/spring-rest/swordv2
+      - swordv2__D__server__P__collection__P__url=http://localhost:8080/spring-rest/swordv2/collection
+      - swordv2__D__server__P__servicedocument__P__url=http://localhost:8080/swordv2/spring-rest/servicedocument
```

Comment:
@terrywbrady : Just a sidenote, if these three URL configurations are necessary for SWORDv2, then I suspect they **also** will be necessary for SWORDv1.  Both SWORD webapps do their own URL validation, and they actively throw 404 errors if a request doesn't match the expected URL *exactly*.  This is something I might be able to cleanup (with more sane default URLs for SWORDv1/v2) in the follow-up PR to https://github.com/DSpace/DSpace/pull/2265, as that follow-up PR will better standardize URLs to all be based off a (new) `dspace.server.url` path.  But, for now, we may need to configure the proper URLs for SWORDv1 and SWORDv2 to work as expected.